### PR TITLE
replace G8OS -> Zero-OS

### PR DESCRIPTION
**Pull request by yveskerwyn**

### CODE REVIEW

File: docs/config/main.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Example:
 fuse_storage = "ardb://hub.gig.tech:16379"
 ```
 
-With `fuse_storage` you set the default key-value store that will be mounted by [G8ufs](../g8ufs/g8ufs.md) when creating containers using the [container.create()](../interacting/commands/corex.md#create) command. The default, as shown above, is the ARDB storage cluster implemented in [G8OS Hub](../g8ufs/hub/hub.md). When creating a new container you can override this default by specifying any other ARDB storage cluster, as documented in [Creating Containers](../containers/creating.md).
+With `fuse_storage` you set the default key-value store that will be mounted by [G8ufs](../g8ufs/g8ufs.md) when creating containers using the [container.create()](../interacting/commands/corex.md#create) command. The default, as shown above, is the ARDB storage cluster implemented in [0-Hub](../g8ufs/hub/hub.md). When creating a new container you can override this default by specifying any other ARDB storage cluster, as documented in [Creating Containers](../containers/creating.md).
```
Comment (grimpy) [inline docs/config/main.md]: this flag has been renamed to storage

Comment (grimpy) [inline docs/config/main.md]: Proably this config should be revised, there is no more redis either

File: docs/containers/creating.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
-We use the [ubuntu1604.flist](https://hub.gig.tech/gig-official-apps/ubuntu1604.flist.md) flist from the [gig-official-apps repository on the G8OS Hub](https://hub.gig.tech/gig-official-apps):
+We use the [ubuntu1604.flist](https://hub.gig.tech/gig-official-apps/ubuntu1604.flist.md) flist from the [gig-official-apps repository on the Zero-OS Hub](https://hub.gig.tech/gig-official-apps):
 
 ![](flist.png)
 
-Here's the Python script using the G8OS client:
+Here's the Python script using the Zero-OS client:
 
 ```python
 import g8core
```
Comment (grimpy) [inline docs/containers/creating.md]: from zeroos.core0.client import Client
cl = Client("IP OF Zero-OS")

File: docs/g8ufs/g8ufs.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # G8ufs
 
-See https://github.com/g8os/g8ufs/blob/1.1.0-alpha/docs/README.md
+See https://github.com/zero-os/0-fs/blob/1.1.0-alpha/docs/README.md
```
Comment (grimpy) [inline docs/g8ufs/g8ufs.md]: this should probably point to the master branch

File: docs/interacting/commands/corex.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ Values:
   - If none it will automatically be set to core-x, x being the ID of the container
 
 - **storage**: URL to the ARDB storage cluster to mount, e.g. `ardb://hub.gig.tech:16379`
-  - If not provided the default one from the G8OS main configuration will be used, see the documentation about `fuse_storage` in [Main Configuration](../../config/main.md) for more details
+  - If not provided the default one from the Zero-OS main configuration will be used, see the documentation about `fuse_storage` in [Main Configuration](../../config/main.md) for more details
```
Comment (grimpy) [inline docs/interacting/commands/corex.md]: fuse_storage has been renamed

File: docs/networking/ovs.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ ovs = int(resp.get().data)
 ```
 
 Note the following:
-- We're using the `ovs.flist` flist here, for more information on how to build it see the [g8os/ovs-plugin](https://github.com/g8os/ovs-plugin) repository on Github
+- We're using the `ovs.flist` flist here, for more information on how to build it see the [Zero-OS/ovs-plugin](https://github.com/zero-os/ovs-plugin) repository on Github
```
Comment (grimpy) [inline docs/networking/ovs.md]: this repo is called openvswitch-plugin now

File: docs/networking/ovs.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ ovscl.json('ovs.vlan-ensure', {'master': 'backplane', 'vlan': 2313, 'name':'vxba
 # the vlan tag can be any reserved value for vxbackend
 ```
 
-> The above calls will make sure we have `backplane` vswitch, and `vxbackend` vswitch but it doesn't connect the backplane to the internet, make sure to check [ovs-plugin](https://github.com/g8os/ovs-plugin) for more info on how to create bonds or add links to the backplane
+> The above calls will make sure we have `backplane` vswitch, and `vxbackend` vswitch but it doesn't connect the backplane to the internet, make sure to check [ovs-plugin](https://github.com/zero-os/ovs-plugin) for more info on how to create bonds or add links to the backplane
```
Comment (grimpy) [inline docs/networking/ovs.md]: reponame has been changed

File: docs/networking/zerotier.md [hunk 7]
```diff
@@ -5,14 +5,14 @@ More info about ZeroTier: https://zerotier.com/manual.shtml
...
 ## How to connect to a ZeroTier network
 
-For this example we assume you have a G8OS node running and you can connect to it.
+For this example we assume you have a Zero-OS node running and you can connect to it.
 
-The following code joins your G8OS node to the public network provided by ZeroTier:
+The following code joins your Zero-OS node to the public network provided by ZeroTier:
 
 ```python
 import g8core
```
Comment (grimpy) [inline docs/networking/zerotier.md]: import for g8core client has changed (see previous comment)

File: docs/config/main.md (hunk 1 above)
Comment (yveskerwyn) [inline docs/config/main.md]: First is updated, for the second I need input

File: docs/containers/creating.md (hunk 2 above)
Comment (yveskerwyn) [inline docs/containers/creating.md]: updated

File: docs/g8ufs/g8ufs.md (hunk 3 above)
Comment (yveskerwyn) [inline docs/g8ufs/g8ufs.md]: yes, and I removed this page, linking all 0-FS topics into the 0-FS repo directly

File: docs/interacting/commands/corex.md (hunk 4 above)
Comment (yveskerwyn) [inline docs/interacting/commands/corex.md]: updated

File: docs/networking/ovs.md (hunk 5 above)
Comment (yveskerwyn) [inline docs/networking/ovs.md]: updated

File: docs/networking/ovs.md (hunk 6 above)
Comment (yveskerwyn) [inline docs/networking/ovs.md]: updated

File: docs/networking/zerotier.md (hunk 7 above)
Comment (yveskerwyn) [inline docs/networking/zerotier.md]: updated

File: docs/networking/ovs.md (hunk 6 above)
Comment (muhamadazmy) [inline docs/networking/ovs.md]: the `ovs-plugin` has been renamed to `openvswitch-plugin` (also repo name)

File: docs/networking/ovs.md (hunk 5 above)
Comment (muhamadazmy) [inline docs/networking/ovs.md]: the `ovs-plugin` has been renamed to `openvswitch-plugin` (also repo name)

**Pull request merged**